kratos/utilities/auxiliar_model_part_utilities.cpp

```diff
@@ -24,6 +24,14 @@
 
 namespace Kratos
 {
+void AuxiliarModelPartUtilities::CopySubModelPartStructure(const ModelPart& rModelPartToCopy)
+{
+    AuxiliarCopySubModelPartStructure(mrModelPart, rModelPartToCopy);
```

Comment:
I think  in this case the `AuxiliarCopySubModelPartStructure` with two arguments is a more clear interface than the `CopySubModelPartStructure` 

At least I would expose both!

Reply:
Thi thing is that AuxiliarModelPartUtilities have a modelpart as input..., I can do an exception here...